kroxylicious-systemtests/pom.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
+        <version>0.3.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>kroxylicious-systemtests</artifactId>
```

Comment:
I think the name kroxylicious-systemtests is too general.  I think this test suite is all about kubernetes, so that should be reflected in the name. kroxylicious-kube-systemtests?

I actually wonder if it is correct to include this module in the kroxylicious project. The reason I say this is that kroxylicious (the operand) should be **completely** unaware of Kubernetes.  The role of this project will be to test kroxylicious, its operator together with strimzi on kubernetes.

So maybe this module should be part of a kroxylicious-operator repo?

Reply:
I think as long as we prevent other things depending on the system tests (an enforcer rule) then I think we defer the decision on which repo for later.